style.css

```diff
@@ -358,3 +392,83 @@ ul.work-list {
 .heading1 {
   margin-top: 245.73px;
 }
+
+form button:active {
+  color: #f4f1f0;
+  background-color: #e05304;
+}
+
+form button:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
+
+form button:hover {
+  border: 1px solid #f55800;
+  color: #f55800;
+  background: #fff5e1;
+}
+
```

Comment:
- [ ] Nice job implementing `hover` interaction styling, Please add `active` interaction styling as well because it's required and you can see it in Figma design.

![image](https://user-images.githubusercontent.com/68796232/160916188-8e53d6a4-4d25-43bd-aa50-9b5cb48836ac.png)